en-us/faq.md

```diff
@@ -0,0 +1,33 @@
+# Vanilla launcher FAQ
+
+## What is this?
+
+- This FAQ gives answers to questions you might have about the vanilla launcher. Please join our [Discord](https://discord.gg/yxaXtaQqdB) and visit the 'support' channel if you have any additional questions. 
+
+### How to install?
+
+- There are many ways to install FO on multiple launchers. Please use this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#minecraft-launcher-vanilla) to install FO on the launcher that you use.
+
+### So I just copy the mods, right?
+
+- No. Fabulously Optimized consists of mods, configs and a resource pack that is built to help the user naviagte through Mod Menu. If you simply copy the mods over, the settings would not be applied, and since you would have to configure the mods yourself, you might not have the proper settings to accurately replicate OptiFine.
+
+### What is the MultiMC ZIP?
+
+- The MultiMC ZIP is a packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge modpacks and downloads from the ZIP file support the creator of FO. This also allows for a better installation of FO for vanilla launcher users.
+
+### What is in the ZIP file?
+
+- In both the CurseForge and MultiMC ZIP files, they contain configs for each mod and resource packs which are essential for FO to run. The config folder contain configs for each mod in FO so the modpack is configured perfectly when you first start up the modpack, the mods folder contains mods that are essential to replicate OptiFine's features, and the resource folder contains a resource pack that helps the user navigate through Mod Menu. There is also a file called 'Copy all 3 folders!', this file makes it easier to update the pack rather than having to download and make a new instance each time there is a new update.
+
+### Why try a different launcher?
+
+- There are a variety of launchers available, each with unique features. If you use the vanilla launcher, you miss out on many features, including the ability to update the modpack and the version of Minecraft, add any mod with a few simple clicks, and finally download the modpack more easily.
+
+### Why is FO not a client?
+
+- FO is not a client because simply we would need to be able to code a client which would require people to know the programming language that we are writing the client in. By being a modpack, this makes it easier for the creator and user to add their own mods in and use Fabulously Optimized as base pack for their own mods they want to add in. This can also make it easier to update the modpack by adding in replacement mods while waiting for other mods in the pack to update.
```

Comment:
incorporate more things from here:
https://github.com/Fabulously-Optimized/fabulously-optimized/issues/101#issuecomment-888805246

programming itself is not the highest concern, rather licensing and integration are
so reword it based on priority and some extra points from the link